apps/src/music/player/MusicPlayer.ts

```diff
@@ -287,9 +285,18 @@ export default class MusicPlayer {
         );
       }
 
+      const folder = library.getFolderForSoundId(soundEvent.id);
+
+      if (folder === null) {
+        this.metricsReporter.logWarning(`No folder for ${soundEvent.id}`);
+        return [];
+      }
+
+      const url = library.generateSoundUrl(folder, soundData);
+
       return [
         {
-          sampleId: soundEvent.id,
+          sampleUrl: url,
```

Comment:
nit: inline this (`sampleUrl: library.generateSoundUrl...`)? unless `url` is being used somewhere else

Reply:
Done.